internal/archive/archive.go

```diff
@@ -28,6 +30,7 @@ type Options struct {
 	Arch       string
 	Suites     []string
 	Components []string
+	Pro        string
```

Comment:
Is adding a function `func (option *Option) isPro() bool` possible? Multiple `Options.Pro == ""` or `Options.Pro != ""` are used. Giving it an `isPro` function can improve the readability.